torchgeo/datasets/millionaid.py

```diff
@@ -0,0 +1,397 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Million-AID dataset."""
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional, cast
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from torchgeo.datasets import VisionDataset
+
+from .utils import check_integrity, download_and_extract_archive, extract_archive
+
+
+class MillionAID(VisionDataset):
+    """Million-AID Dataset.
+
+    The `MillionAID <https://captain-whu.github.io/DiRS/>`_ dataset consists
+    of one million aerial images from the Google Earth Engine that offers
```

Comment:
```suggestion
    of one million aerial images from Google Earth Engine that offers
```
GEE is a proper noun, no need for "the"